File: AD/Hospital_JPA_Raro/src/main/java/hospitaljpa/dao/model/MedicalRecord.java
package hospitaljpa.dao.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class MedicalRecord {
    private int id;
    private int idPatient;
    private int idDoctor;
    private String diagnosis;
    private LocalDate date;
    private List<Medication> medications;

    public MedicalRecord(int id, int idPatient, int idDoctor, String diagnosis, LocalDate date) {
        this.id = id;
        this.idPatient = idPatient;
        this.idDoctor = idDoctor;
        this.diagnosis = diagnosis;
        this.date = date;
    }
}

File: Moviles/AppTareasCompose/app/src/main/java/com/example/apptareascompose/domain/model/MedicalRecord.kt
package com.example.apptareascompose.domain.model

import java.time.LocalDate

data class MedicalRecord(
    val id : Int = 0,
    val description : String = "",
    val date : LocalDate = LocalDate.now(),
    val patientId : Int = 0,
    val doctorId : Int = 0,
    val medications : List<String> = emptyList(),
)
